dockerfiles/Dockerfile.todo_app

```diff
@@ -0,0 +1,14 @@
+FROM python:3.10.12 AS BASE
```

Comment:
This file looks great! 😄 

One thing that could be worth thinking about, is whether we want to pin the patch version here; for example if we used `python:3.10` instead of `python:3.10.12` then when Python releases a patch to `3.10.13` we'll pick that up automatically which is nice for simple automated updates, but it does introduce some risk that we change versions and there's an uncaught breaking change. There's no simple answer here, but worth considering that trade off!

Reply:
Thanks for this & I will make the change.